nativelib/src/main/resources/types.h

```diff
@@ -1,18 +1,28 @@
 #ifndef __TYPES_H
 #define __TYPES_H
 
-typedef unsigned long scalanative_dev_t;
-typedef unsigned int scalanative_mode_t;
-typedef unsigned long long scalanative_ino_t;
-typedef unsigned int scalanative_uid_t;
-typedef unsigned int scalanative_gid_t;
-typedef long long scalanative_off_t;
-typedef long int scalanative_time_t;
-typedef long long scalanative_blkcnt_t;
-typedef long scalanative_blksize_t;
-typedef unsigned long scalanative_nlink_t;
+// this is cross-patform definition of integer types
+typedef signed char scalanative_int8_t;
+typedef short scalanative_int16_t;
+typedef int scalanative_int32_t;
+typedef long long scalanative_int64_t;
+typedef unsigned char scalanative_uint8_t;
+typedef unsigned short scalanative_uint16_t;
+typedef unsigned int scalanative_uint32_t;
+typedef unsigned long long scalanative_uint64_t;
```

Comment:
I don't mind actually, but we will partly lose windows 7 support then. It's still possible to have it to compile on win7 but it will require additional installation steps and latest windows update.